python/ray/tests/test_scheduling_strategy.py

```diff
@@ -0,0 +1,87 @@
+import sys
+import pytest
+
+import ray
+import ray.cluster_utils
+from ray.util.client.ray_client_helpers import connect_to_client_or_not
+from ray.util.scheduling_strategies import PlacementGroupSchedulingStrategy
+
+
+@pytest.mark.parametrize("connect_to_client", [True, False])
+def test_placement_group_strategy(ray_start_cluster, connect_to_client):
```

Comment:
Why don't we just add all strategy tests to `test_scheduling` instead of making a new test suite?